json-spec/json-schema/stac-item.json

```diff
@@ -25,21 +25,10 @@
           "required": [
             "id",
             "type",
-            "geometry",
             "links",
             "assets"
           ],
           "properties": {
-            "geometry": {
```

Comment:
I think removing the geometry property altogether is wrong. It should be still required, but the enum should be extended to all types supported by GeoJSON.

Reply:
This only removes the geometry from the STAC Item schema, it still exists in the GeoJSON schema and it is still required that a STAC item be a GeoJSON object.

Reply:
We only had the geometry defined in the STAC item schema to hold the polygon/multipolygon only restriction.  Now that that restriction is removed, we can remove the redundant geometry definition.

Reply:
Oh, thanks for the clarification. Didn't know that we inherit from the GeoJSON spec.